### PR TITLE
Feature/tolerations and node selector inputs

### PR DESCRIPTION
**Pull request by sepulworld**
Signed-off-by: Zane Williamson <zanew@zillowgroup.com>

**Describe your changes**
Extending the Custom Options API to include podOptions to support Tolerations and Node Selector inputs for both SolrCloud StatefulSet and PromExporter Deployment

Updated readme NAMESPACE value to include forward slash

**Testing performed**
Added unit test to validate found and expected Tolerations and Node Selector values

Tested in local docker and dev environment

### CODE REVIEW

File: example/test_solrcloud.yaml [hunk 1]
```diff
@@ -14,13 +14,14 @@ spec:
...
     tag: 8.2.0
   solrJavaMem: "-Xms1g -Xmx3g"
-  solrPodPolicy:
-    resources:
-      limits:
-        memory: "1G"
-      requests:
-        cpu: "65m"
-        memory: "156Mi"
+  customSolrKubeOptions:
```
Comment (HoustonPutman) [inline example/test_solrcloud.yaml:17]: Thanks for making this fix, I had a branch that made these changes but I forgot to make a PR.

**Comment by sepulworld** [comment]
> We can make the same changes for the SolrPrometheusExporter. I'd look in these two places:
> 
> https://github.com/bloomberg/solr-operator/blob/master/controllers/util/zk_util.go#L383
> https://github.com/bloomberg/solr-operator/blob/master/controllers/util/prometheus_exporter_util.go#L219



> We can make the same changes for the SolrPrometheusExporter. I'd look in these two places:
> 
> https://github.com/bloomberg/solr-operator/blob/master/controllers/util/zk_util.go#L383
> https://github.com/bloomberg/solr-operator/blob/master/controllers/util/prometheus_exporter_util.go#L219

I did for prometheus exporter already: https://github.com/bloomberg/solr-operator/pull/107/files#diff-50635f351caf6e78f4f33fec707bd8f4 was there another change for this service I am missing?

Ill look at the zk_util.go and add there.

**Comment by sepulworld** [comment]
> > We can make the same changes for the SolrPrometheusExporter. I'd look in these two places:
> > https://github.com/bloomberg/solr-operator/blob/master/controllers/util/zk_util.go#L383
> > https://github.com/bloomberg/solr-operator/blob/master/controllers/util/prometheus_exporter_util.go#L219
> 
> > We can make the same changes for the SolrPrometheusExporter. I'd look in these two places:
> > https://github.com/bloomberg/solr-operator/blob/master/controllers/util/zk_util.go#L383
> > https://github.com/bloomberg/solr-operator/blob/master/controllers/util/prometheus_exporter_util.go#L219
> 
> I did for prometheus exporter already: https://github.com/bloomberg/solr-operator/pull/107/files#diff-50635f351caf6e78f4f33fec707bd8f4 was there another change for this service I am missing?
> 
> Ill look at the zk_util.go and add there.

Ok updated zk_util.go

**Comment by HoustonPutman** [comment]
Ah can't believe I missed that. I'll try to run this today and try it out! The code looks good to me though

**Comment by sepulworld** [comment]
No problem! Thanks for taking a look! I added a couple tests too

**Pull request merged**